source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -416,6 +416,36 @@ Server error, status code: 409, error code: 60016, message: An operation for ser
 
 This happens because you cannot do anything to a service instance while it's in a pending state. A CDN service instance stays pending until it detects the CNAME or ALIAS record. If this causes a problem for you, [contact support](https://www.cloud.service.gov.uk/support) to ask us to manually delete the pending instance.
 
+#### View the service configuration
+The `message` field may appear blank after certain operations. In that case, you can get information about the configured domains using:
```

Comment:
What's the reason it appears blank? Does this happen randomly or is it for some operations and not others?

Reply:
It appears blank after a synchronous operation, such as changing the forwarded headers. It's a missing piece/failure of the OSBAPI spec and how CloudFoundry implements it.

Reply:
I think it might be worth adding it in, unless you think it could confuse the reader?

Reply:
I don't think I'd want to explain the _why_, because I don't think users should need to know. I'll update it to say "appear blank after operations like changing the forwarded headers" though.